### DISCUSSION TITLE
Is there a way to choose the color mode on a per slide basis?

### DISCUSSION THREAD

**Discussion by remiconnesson** [slidevjs/slidev#1211] [Q&A]
Let's say that I want to use light mode for most of my slides, but on some slides I'd like to use a very dark background color/image.

Is there a way to set the color mode on a per slide basisI

I saw : https://github.com/slidevjs/slidev/discussions/1087#discussioncomment-6936556 but it doesn't seem to work on a per slide basis.

**Comment by dvasiliu** [comment]
I have the same question and more: how could I change the color of one slide? Apparently, it is very easy to change the color of the ink for the text, but the background color isn't an option that I could figure out.

**Comment by kermanx** [comment]
It is not easy to set the color scheme for one slide because of the limitation of CSS selector. However, I think most of the effects can be achieved via custom CSS. You can inspect the DOM tree via the browser's DevTool and add your custom CSS to `./styles.css`
